lib/helpers/donation_helpers.dart

```diff
@@ -0,0 +1,44 @@
+enum DonationState {
```

Comment:
I would suggest to rename the file as donation_state.dart and move all functions to the enum itself